### PR TITLE
Add rudimentary toolset

### LINKED ISSUE #1: Setup project
Setup new project with rudimentary tools:

- Rails 7
- Ruby 3
- Rubocop
- FactoryBot
- RSpec
- Simplecov

### PR DESCRIPTION
**Pull request by katemildner**
All the basic tools are installed and configured, ready to go.

### CODE REVIEW

**Review by ktopolski — APPROVED** [review]
Checked all of the essential commands and they run successfully. No warnings, no errors. Well done!

**Comment by ktopolski** [comment]
Although I can't merge this PR @katemildner - I don't have write access to this repo :D

**Comment by katemildner** [comment]
> Although I can't merge this PR @katemildner - I don't have write access to this repo :D

I think I've found the cause of that, but I allowed myself to merge the PR, just this one time ;)

**Pull request merged**
